tests/e2e/mnist.py

```diff
@@ -151,7 +170,7 @@ def test_dataloader(self):
 trainer = Trainer(
     accelerator="auto",
     # devices=1 if torch.cuda.is_available() else None,  # limiting got iPython runs
-    max_epochs=3,
+    max_epochs=5,
```

Comment:
Is there any reason to have more epochs? It will probably slow down the test.

Reply:
Actually, not sure about this.. it was recently updated in mnist_disconnected.py in guided demos dir :eyes: 
Thanks @jiripetrlik for reviewing !

Reply:
@jiripetrlik I have updated it to previous version to use 3 epochs :+1: